specification/components/stu3/schemas/endpoints/a006-retrieve-attachment.yaml

```diff
@@ -11,19 +13,31 @@ description: |
 
 
   ## Pre-requisites
-  ### Application-restricted access
-  In order to use this endpoint you must be an authenticated e-RS calling application, working in the context of a Service Provider Organisation.
+  ### Authentication
+  #### Application-restricted access
+  In order to use this endpoint you must be an authenticated e-RS calling application.
 
-  ### Healthcare worker, user-restricted access
+  #### Healthcare worker, user-restricted access
   In order to use this endpoint you must be an authenticated e-RS user or application and use one of the following e-RS roles:
     - `REFERRING_CLINICIAN`
     - `REFERRING_CLINICIAN_ADMIN`
     - `SERVICE_PROVIDER_CLINICIAN`
     - `SERVICE_PROVIDER_CLINICIAN_ADMIN`
 
-  You need to have retrieved the referral via [[HYPERLINK_A005]] to obtain the URLs for its attachments.
+  ### Attachment availability
+  In order to use this endpoint, the attachment to be retrieved must be available for download. Attachments are only made available once an uploaded file has been validated and successfully scanned for malware. A request to retrieve an attachment that is not available for download will result in a 422 error being returned. See the Response section for further information.
+
+  The availability status of an attachment can be retrieved via any endpoint that provides details of an attachment in the success response. Details of the availability statuses that may be returned via these endpoints can be found in the specification for [[HYPERLINK_A005]].
+
+  Prior to retrieving an attachment, you will need to have retrieved the referral or advice request the attachment is associated with (via the [[HYPERLINK_A005]] or [[HYPERLINK_A024]], for example). Referrals and advice requests include resolvable URLs to the files currently attached to them, which can be used with this endpoint to retrieve the attachments themselves.
```

Comment:
I think in line 32 is missing the word endpoints before "for example)":

"endpoints, for example)"